src/Apps/Order/Components/ApplePayDetails.tsx

```diff
@@ -9,6 +10,8 @@ export const ApplePayDetails = (props: Props) => {
 
   return (
     <Flex alignItems="center">
+      <ApplePayMarkIcon mr={1} width="26px" height="26px" />
+
       <Text
```

Comment:
#nit-pick: I was curious why the text was 1px off the center line (while the icon was centered), and I realized there is this `position: "relative", top: "1px"` styling for the text below (same for the [credit card details](https://github.com/artsy/force/blob/8503087deec62a7f5fb6e027163d96c2b5ab68d4/src/Apps/Order/Components/CreditCardDetails.tsx#L32)). Not sure why we added the 1px top but perhaps it's okay to remove?

![Screenshot 2025-04-16 at 9 08 32 AM](https://github.com/user-attachments/assets/9870f334-cb41-4ddd-9e4c-6ccbe33eaaec)

Reply:
🦅 
I'll remove it and check

Reply:
💯  Appreciate all the screenshots!